### DISCUSSION TITLE
Online solo score claims not to be downloaded

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#26619] [Q&A] [answered]
Noticed something weird today, which resembles #24967 but seems to have lighter symptoms.
I don't know what state that OP score is in (has it been forgotten?), but mine is not multiplayer-related. Just solo.

I was looking at some of my replays when I saw [this one](https://lazer.ppy.sh/scores/2448482645) again, from 3 months ago.

By checking the scores panel list, my score claims not to be downloaded when in fact it is:
![image](https://github.com/ppy/osu/assets/43073074/dd61089b-009a-4df0-8843-d9e86ffb0425)
![image](https://github.com/ppy/osu/assets/43073074/f69f0e04-1e85-4fe9-8a80-788c17962347)

Attempting to download it...seems to be successful, but the button remains stuck in a downloading state (as with the OP of the linked issue).
These things happen:
- a notification claims to have downloaded it
- the `database` log is quite interesting:
  ```
  2024-01-18 13:16:30 [verbose]: [?????] Beginning import from tmp57FA.osr...
  2024-01-18 13:16:31 [verbose]: [8ad79] Import successfully completed!
  2024-01-18 13:16:50 [verbose]: [?????] Beginning import from tmpA65A.osr...
  2024-01-18 13:16:50 [verbose]: [8ad79] Found existing but failed re-use check.
  2024-01-18 13:16:50 [verbose]: [8ad79] Import successfully completed!
  2024-01-18 13:26:22 [verbose]: [?????] Beginning import from tmp5DCD.osr...
  2024-01-18 13:26:22 [verbose]: [8ad79] Found existing but failed re-use check.
  2024-01-18 13:26:22 [verbose]: [8ad79] Import successfully completed!
  ```
  the first time it claims to just have imported it correctly (even though it was actually already downloaded). But if I re-download it from the score list (which keeps thinking it doesn't have the downloaded score), it also starts printing `Found existing but failed re-use check.`

If I open the local score, it's shown separately (left one is local):
![image](https://github.com/ppy/osu/assets/43073074/97731f06-b458-4cab-aca9-4c98cf8a7bdf)

It seems quite strange this situation.
I don't know what else to say, if you need more info please ask!

**Comment by Gabixel** [comment]
Actually I just noticed those 2 local scores after trying to download the online one, which seem to have a slightly different accuracy for some reason...? (The online score still claims not to be downloaded)

![image](https://github.com/ppy/osu/assets/43073074/ec28e771-dd2e-4a5a-aa65-f34e1ccdabd2)

**Comment by bdach** [comment, marked as answer]
Any and all lazer online replays recorded before November 12 can be arbitrarily broken for a whole mess of reasons. This will not be retroactively fixed because it cannot be (there is no data to do this correctly in all cases).

If you find this happening with _newer_ scores than that then please report again.
